services/mailer/mail.go

```diff
@@ -310,7 +310,13 @@ func composeIssueCommentMessages(ctx *mailCommentContext, lang string, recipient
 
 	msgs := make([]*Message, 0, len(recipients))
 	for _, recipient := range recipients {
-		msg := NewMessageFrom(recipient.Email, ctx.Doer.DisplayName(), setting.MailService.FromEmail, subject, mailBody.String())
+		msg := NewMessageFrom(
+			recipient.Email,
+			fmt.Sprintf("%s (@%s)", ctx.Doer.DisplayName(), ctx.Doer.Name),
```

Comment:
I think this value should be offered as a method in `/models/user/user.go` instead.
Perhaps something like `GetCompleteName`?
That way, it can be used by other parts as well.